src/style.css

```diff
@@ -1,7 +1,332 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat+Alternates:ital,wght@0,400;0,800;0,900;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro&display=swap');
+
+* {
+  text-decoration: none;
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+
+  /* outline: 2px solid red; */
+  font-size: 20px;
+}
+
+:root {
+  --color-primary: #fa3496;
+  --color-secondary: #f6f6f6;
+  --color-third: #1c1e29;
+}
+
 body {
-  background-color: lightblue;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  background-image: linear-gradient(180deg, #040506, #0c0d12, #1c1e29, #34384c);
+  background-blend-mode: soft-light;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  backdrop-filter: blur(10px);
+  font-family: 'Source Sans Pro', sans-serif;
+}
```

Comment:
- [ ] Your page design does not fit the desktop screen size. Please try to work more on your design so it fits all the desktop screen sizes. Try to make it flexible as much as you can. Ensure that the page is not scrolling horizontally.
- Your design:
<img width="487" alt="image" src="https://user-images.githubusercontent.com/77190203/181995526-8db5e41d-689d-44db-ac0a-df85b62a2e97.png">

Reply:
Fixed